ConfigSamples/Smoothieboard/config

```diff
@@ -109,6 +109,17 @@ laser_module_pwm_pin                          2.5             # This pin will be
                                                               # the maximum and minimum power levels specified above
 #laser_module_pwm_period                      20              # This sets the pwm frequency as the period in microseconds
 
+## Motor driver configuration
```

Comment:
This needs to be removed from the standard config and moved to the Snippets under its own file, please.

Reply:
Done